src/components/CountryBoard.vue

```diff
@@ -54,23 +53,22 @@ onMounted(() => {
         </div>
       </div>
     </div>
-    <span v-else class="c-countryBoard__loader">LOADING</span>
+    <p v-else class="c-countryBoard__loader">LOADING</p>
```

Comment:
To raczej powinien być div/span nie paragraf tekstu. Jak chcesz być semantycznie poprawny to znalazłem takie coś
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role